mapping/sustainability.yaml

```
@@ -103,8 +208,48 @@
     module: Economic and fiscal
     indicator: Multi-year implementation
     disclosure format: 'In case of multiyear project implementation, disclose information on budget projection for all years of implementation (E.g. Y1: \[currency and amount\], Y2: \[currency and amount\], etc).'
-    mapping: ''
-    example: ''
+    mapping: |-
+        Project level:
+
+        1. Add a `BudgetBreakdowns` object to the `budget.budgetBreakdowns` array.
+
+        2. Set it's `.id` incrementally and it's `.description` to "Breakdown by year of implementation".
+
+        3. For each year of implementation, add a `BudgetBreakdown` object to the `budget.budgetBreakdowns.budgetBreakdown` array. For each object:
+
+        - Set the `.id` incrementally.
+        - Set the `.description` to the year of implementation.
+
+        4. For each year, map the amount and currency to the budget breakdown's `.amount`
+    example: |-
+        {
+          "budget": {
+            "budgetBreakdowns": [
+              {
+                "id": "1",
+                "description": "Breakdown by year of implementation",
+                "budgetBreakdown": [
+                  {
+                    "id": "1",
+                    "description": "2024",
```

See updates to mapping

```suggestion
                    "period": {
                      "2024-01-01T00:00:00Z",
                      "2024-12-31T00:00:00Z"
                    },
```